app/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+CONFIG_ZMK_USB=y
+CONFIG_ZMK_BLE=y
+
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+CONFIG_NVS=y
+CONFIG_SETTINGS_NVS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+
+# Default main stack size is too small for HCI RPMsg
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
This bit is probably not necessary for the nrf52840 DK. It doesn't use HCI RPMsg since it only has one CPU.

Reply:
Thanks. Missed that when I copied it over.